### ISSUE TITLE
Bad programming custom.

### ISSUE THREAD

**Issue by tbwork** [openzipkin/zipkin#2046]
In terms of Maven in Eclipse, two same-name artifactIds can not be imported in one workspace.
Also as many other users mentioned, both zipkin and zipkin2 are packaged as zipkin-{version}.jar.
This is very annoying to the fresh of Zipkin.

**Comment by codefromthecrypt** [comment]
You are the only one to report this eclipse problem, even if we did change something, we'd need verification as I'm very surprised eclipse doesn't work. We have core committers who use eclipse.

WRT workspace, we will continue to have both versions released until such time as we don't require both. Our collector code currently requires both libraries for interop. When I get back from holiday (21 May), I will start working on removing the last dependencies on v1.

I'll mention point-by-point why there will be no action apart from removing the v1 code as soon as we can (which is at any rate a goal).

okhttp is an example of another library which only differs on group ID and package name. It is one of the most popular libraries out there. From a packaging POV we are no different (ex okhttp.jar is for all versions). What's different between us and them is that we still release 2 packages (because we must! not because we are intentionally being annoying)

Most users never have to import our source in the first place. Often times people are importing code, trying to make custom servers (which is unsupported). The larger audience of Zipkin are not affected by this in any way, as there is no such practice in wider repositories such as brave or zipkin-reporter.

TL;DR: I'm closing this out as it will be addressed by removing the v1 library... to conserve hands for larger benefit and cause least disruption while accomplishing the same goal.

Meanwhile, I'd recommend joining gitter to chat with folks about whatever you are importing the code. You might not even need to be doing this.. Not checking out this repo is another way to not be annoyed by it! :D
https://gitter.im/openzipkin/zipkin

Finally, I understand your frustration and appreciate your telling us about it. Closing this issue doesn't mean we won't address the concern. It is just that we won't do it by reorganizing our group IDs etc. It will be by removing the codebase in question as soon as we can. Ideally, but no promises.. in june.

**Comment by codefromthecrypt** [comment]
see https://github.com/openzipkin/zipkin/issues/2047

**Comment by tbwork** [comment]
@adriancole Thanks for your response. I understood your original intention. Thanks again for providing such a good app tracing tool. Good night and keep healthy!
